include/macros.h

```diff
@@ -51,6 +51,7 @@
 
 #define CLOCK_TIME(hr, min) (s32)(((hr) * 60 + (min)) * 0x10000 / (24 * 60))
 #define CLOCK_TIME_MINUTE  (CLOCK_TIME(0, 1))
```

Comment:
I wonder if this should be `MINUTE_LENGTH` to match `DAY_LENGTH`, depends how it's used

Reply:
Seems like this one could use some discussion, and also seems like scope creep for what was intended for this PR. I'm still not 100% sold on DAY_LENGTH as well but it's better than not having it and it can be changed later